### PR TITLE
fix: Add environment variables validation and tips

### PR DESCRIPTION
**Pull request by weili520**
Signed-off-by: TheYoungManLi <cjl@kubesphere.io>

### What type of PR is this?
/kind bug

### Which issue(s) this PR fixes:
Fixes ##2790

### Special notes for reviewers:

https://user-images.githubusercontent.com/33231138/146139315-cd82dfd5-a89f-4066-8026-ad99c2ebadfd.mov

### Does this PR introduced a user-facing change?
```release-note
Add environment variables validation and error prompt in the container edit page.
```

### CODE REVIEW

**Comment by zryfish** [comment]
I think this PR does introduce a user-facing change, so a release note is required, also /cc @kubesphere/sig-docs , please help review the documentation.

**Comment by weili520** [comment]
> I think this PR does introduce a user-facing change, so a release note is required, also /cc @kubesphere/sig-docs , please help review the documentation.

File: locales/en/l10n-projects-applicationWorkloads-deployments-list.js [hunk 1]
```diff
@@ -34,6 +34,11 @@ module.exports = {
   QUOTA_OVERCOST_TIP:
     'The current resource occupation has exceeded the remaining',
 
+  // List > Create > Pod Settings > Add Container > Container Settings > Environment Settings
+  ENVIRONMENT_INVALID_TIP:
+    "A valid environment variable name must consist of alphabetic characters, digits, '_', '-', or '.', and must not start with a digit.",
```
Comment (Felixnoo) [inline locales/en/l10n-projects-applicationWorkloads-deployments-list.js:39]: `A valid environment variable name must consist of alphabetic characters, digits, '_', '-', or '.', and must not start with a digit.`
->
`The key of an environment variable can contain only letters, numbers, underscores (_), hyphens (-), and periods (.), and must not start with a number.`

File: locales/en/l10n-projects-applicationWorkloads-deployments-list.js [hunk 2]
```diff
@@ -34,6 +34,11 @@ module.exports = {
...
+  ENVIRONMENT_CANNOT_BE_EMPTY: "Environment variable's key cannot be empty.",
```
Comment (Felixnoo) [inline locales/en/l10n-projects-applicationWorkloads-deployments-list.js:40]: `Environment variable's key cannot be empty.`
->
`Please set a key for the environment variable.`

**Review by Felixnoo — CHANGES_REQUESTED** [review]
Please see my comments, thanks.
/cc @Patrick-LuoYu

File: locales/en/l10n-projects-applicationWorkloads-deployments-list.js (hunk 2 above)
Comment (weili520) [reply locales/en/l10n-projects-applicationWorkloads-deployments-list.js]: Thanks.

**Comment by weili520** [comment]
> /hold

I need to change the error tip style.

**Pull request merged**